bot/resources/tags/regex.md

```diff
@@ -0,0 +1,15 @@
+**Regular expressions**
+Sometimes you might want to detect a certain pattern of characters in a string without looking for every possible substring. Regular expressions (regex) are a language for specifying patterns that is used in many programming languages. Python's standard library includes the `re` module, which defines functions for using regex patterns.
```

Comment:
I would prefer something shorter and less formal:

> Regular expressions (regex) are a tool for finding patterns in strings. The standard library's `re` module defines functions for using regex patterns.

What you wrote is good, but many of our users are young and/or not fluent in English. Also, novice developers might see that regex are a *language* and think that learning to use them is like learning a new programming language.

It's difficult to explain what regex can do without using formal terminology, so let's keep the intro brief.

Reply:
Good points, changed 👍